cmd/worker/seed.go

```diff
@@ -0,0 +1,25 @@
+package worker
+
+import (
+	"context"
+	_ "embed"
```

Comment:
@adtac Should the underscore be added for other imports of "embed"? Since we aren't really using it apart from side effects

Reply:
No, we're actually using all the other imports in the source code, but `embed` is special. See https://pkg.go.dev/embed for details (tldr: we need the import so that the `seed.sql` file gets embedded into the binary, but we don't actually need the embed package anywhere in `seed.go`).

Reply:
@adtac I meant to say that other places where we're importing "embed" should do the same thing, but we don't use the underscore there (eg: clickhouse.go)

Reply:
Oop never mind, looks like we are actually

Reply:
Yep, we sometimes need to use the package (e.g. `embed.FS`), so in such files, it's not an anonymous import. The Go compiler doesn't let you compile with unused imports anyway :)